### PR TITLE
Minor README changes for cohort 2024 folder

### PR DESCRIPTION
**Pull request by bbrewington**
Changes:
1. Because the diagram changed in this commit: https://github.com/DataTalksClub/data-engineering-zoomcamp/commit/be68361c402995a8057ab9f70e930f7c56c18ec9 - collapsed the two now redundant links into one
2. Reformatted README's so they're more explicit than implicit (currently, pretty hard to follow)
    1. [01-docker-terraform/README.md](https://github.com/DataTalksClub/data-engineering-zoomcamp/tree/main/01-docker-terraform/README.md)
    2. [cohorts/2024/README.md](https://github.com/DataTalksClub/data-engineering-zoomcamp/tree/main/cohorts/2024/README.md)

### CODE REVIEW

File: 01-docker-terraform/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ### Introduction
 
-* [Video](https://www.youtube.com/watch?v=-zpVha7bw5A)
+* [Video: "Data Engineering Zoomcamp 2023"](https://www.youtube.com/watch?v=-zpVha7bw5A)
```
Comment (alexeygrigorev) [inline 01-docker-terraform/README.md:3]: Perhaps it's also worth adding 2024 launch video here? What do you think?

Comment (bbrewington) [reply 01-docker-terraform/README.md:3]: Good call - done! (I added couple other links as well - see most recent commit)

**Comment by alexeygrigorev** [comment]
It appears there's a conflict... Would it be possible to resolve it?

**Comment by alexeygrigorev** [comment]
I'll close it for now, feel free to reopen
